src/constants.js

```diff
@@ -21,7 +22,10 @@ const STAGE_ENVIRONMENTS = Object.freeze({
 });
 
 const IETF_TAGS = Object.freeze([ENGLISH_IETF_TAG, SPANISH_IETF_TAG]);
-const LANGUAGE_CODES = Object.freeze([ENGLISH_LANGUAGE_CODE, SPANISH_LANGUAGE_CODE]);
+const LANGUAGE_CODES = Object.freeze([
+  ENGLISH_LANGUAGE_CODE,
+  SPANISH_LANGUAGE_CODE,
+]);
 
 const IETF_TAGS_TO_CONTAINER_ROLE_LABEL = Object.freeze({
   [ENGLISH_IETF_TAG]: 'Notice about use of cookies on edx.org.',
```

Comment:
I don't know what this is for, but does it need to be generalized away from "edx.org"?